ghcide/src/Development/IDE/Core/Shake.hs

```diff
@@ -742,6 +739,18 @@ instantiateDelayedAction (DelayedAction _ s p a) = do
       d' = DelayedAction (Just u) s p a'
   return (b, d')
 
+mRunLspT :: Applicative m => Maybe (LSP.LanguageContextEnv c ) -> LSP.LspT c m () -> m ()
+mRunLspT (Just lspEnv) f = LSP.runLspT lspEnv f
+mRunLspT Nothing _ = pure ()
```

Comment:
So when invoking ghcide from the cli the `LanguageContextEnv` should be null right?

Reply:
Yeah